diff --git a/node_modules/@react-native-firebase/analytics/lib/index.js b/node_modules/@react-native-firebase/analytics/lib/index.js
index 581af58..c05fb39 100644
--- a/node_modules/@react-native-firebase/analytics/lib/index.js
+++ b/node_modules/@react-native-firebase/analytics/lib/index.js
@@ -25,7 +25,7 @@ import {
   isOneOf,
   isString,
   isUndefined,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 
 import {
   createModuleNamespace,
@@ -33,7 +33,7 @@ import {
   getFirebaseRoot,
 } from '@react-native-firebase/app/lib/internal';
 import { setReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
-import { isBoolean } from '@react-native-firebase/app/lib/common';
+import { isBoolean } from '@react-native-firebase/app/lib/common/index.js';
 
 import { validateStruct, validateCompound } from './struct';
 import fallBackModule from './web/RNFBAnalyticsModule';
